schemas/development/mdsf.schema.json

```diff
@@ -33,6 +33,14 @@
         "$ref": "#/definitions/MdsfFormatter_for_Tooling"
       }
     },
+    "line_endings": {
```

Comment:
Why is there an S at endings here, and not everywhere?

Reply:
"line endings" is the plural form of "line ending", i think 😅

Reply:
But it could just as well be called `newline` instead.

Reply:
Actually, I think changing it to `newline` would make it clearer

Reply:
Changed in https://github.com/hougesen/mdsf/pull/996